ddtrace/contrib/vertica/patch.py

```diff
@@ -75,32 +77,36 @@ def cursor_span_end(instance, cursor, _, conf, *args, **kwargs):
             "vertica_python.vertica.cursor.Cursor": {
                 "routines": {
                     "execute": {
-                        "operation_name": "vertica.query",
+                        "operation_name": schematize_database_operation("vertica.query", database_provider="vertica"),
                         "span_type": SpanTypes.SQL,
                         "span_start": execute_span_start,
                         "span_end": execute_span_end,
                         "measured": True,
                     },
                     "copy": {
-                        "operation_name": "vertica.copy",
+                        "operation_name": schematize_database_operation("vertica.copy", database_provider="vertica"),
```

Comment:
Noting `copy` and `nextset` spans will now add into the `vertica.query` metrics. Does this seem right?

Reply:
I wonder if "query" is the right name.... Does it count as querying if you're writing to a database? Feel free to ignore, just thinking.

Reply:
No, this is a good catch, I'm taking copy out